WNPRC_EHR/src/org/labkey/wnprc_ehr/pages/query/actions.ts

```diff
@@ -0,0 +1,122 @@
+//Returns most recent demographics given animal id
+import { ConfigProps } from "../typings/main";
+import {Query, ActionURL} from "@labkey/api";
+
+
+interface jsonDataType {
+    commands: object;
+}
+
+//Expects json row data with commands property, see API docs for saveRows
+export function saveWeightActionSaveRowsDirect(jsonData: jsonDataType) {
+    return new Promise((resolve, reject) => {
+
+
+        /* Query.saveRows({
+           commands: jsonData.commands,
+           method: "POST",
+           containerPath: ActionURL.getContainer(),
+           success: data => {
+             console.log("success");
+             resolve(data);
+           },
+           failure: data => {
+             console.log("error");
+             console.log(data);
+             reject(data);
+           }
+         });*/
```

Comment:
Commented out code, but looks like saveWeightActionSaveRowsDirect() isnt being used

Reply:
@csebranek Can you look into this?